addons/crm_livechat/models/crm_lead.py

```diff
@@ -1,23 +1,31 @@
 from odoo import api, fields, models
+from odoo.exceptions import AccessError
 
 
 class CrmLead(models.Model):
     _inherit = "crm.lead"
 
-    channel_id = fields.Many2one(
+    livechat_channel_id = fields.Many2one(
```

Comment:
This should be `discuss_channel_id` if you need the name to be more specific. livechat channel should be reserved for `im_livechat.channel` to avoid confusion.

Reply:
Purpose is to ease grep, so maybe ``original_channel_id`` which is functionally more precise.